client/src/components/LoadingScreen.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Gif from "../assets/loading_animation.gif";
```

Comment:
Consider renaming this import to clarify that you only receive a path/string.

```suggestion
import loadingAnimationSrc from "../assets/loading_animation.gif";
```